### PR TITLE
added manpage from debian package

### LINKED ISSUE #1072: no-manual-page-for-binary lmp
rpmlint is complaining that `lmp` has no manual page, something for the next stable release.

### PR DESCRIPTION
**Pull request by junghans**
Closes #1072

### CODE REVIEW

File: doc/lammps.1 [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.TH LAMMPS "2012-02-23"
```
Comment (rbberger) [inline doc/lammps.1]: probably should update this date and use the release date?

File: doc/lammps.1 [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Atomic/Molecular Massively Parallel Simulator. LAMMPS has potentials for soft 
+materials (biomolecules, polymers) and solid-state materials (metals, 
+semiconductors) and coarse-grained or mesoscopic systems. It can be used to 
+model atoms or, more generically, as a parallel particle simulator at the 
+atomic, meso, or continuum scale.
+
+See http://lammps.sandia.gov/ for documentation.
+
+.SH COPYRIGHT 
+© 2003--2012 Sandia Corporation
```
Comment (rbberger) [inline doc/lammps.1]: same here 2018

**Comment by rbberger** [comment]
@junghans should we add an install to cmake to put it in the right place?

**Comment by junghans** [comment]
@rbberger: install rule added

**Comment by rbberger** [comment]
great! looks good

**Comment by junghans** [comment]
@akohlmey good point, I changed the installation name of the manpage. I also changed to `-in` syntax as suggested.

However, what options should actually go into the manpage, I will leave to @sjplimp.

**Comment by sjplimp** [comment]
@junghans all the command-line options are on this page:  doc/src/Run_options.txt.  You can use
whatever info there you like for this man page

**Comment by rbberger** [comment]
rerunning checks, was switching to an additional build server but that went wrong...

**Pull request merged**